OutOfSchool/OutOfSchool.RazorTemplatesData/Services/EmailTemplates.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace OutOfSchool.RazorTemplatesData.Services
+{
+    public static class EmailTemplates
+    {
+        /// <summary>
+        /// Bass path to the html email templates
+        /// </summary>
+        private static string MainPath => "/Views/Emails/";
+
+        /// <summary>
+        /// Bass path to the plain text email templates
+        /// </summary>
+        private static string MainPathPlainText => "/ViewsPlainText/Emails/";
+
+        /// <summary>
```

Comment:
This comment block is related only to first property, while is describing 4 properties, please rewrite it in simple `// ` comment, or add personal comments (though I see no necessity in commenting them at all)